package.json

```diff
@@ -232,7 +232,7 @@
         "react-dom": "^16.9.0",
         "react-draggable": "^3.3.0",
         "react-immutable-proptypes": "^2.1.0",
-        "react-intl": "^2.8.0",
+        "react-intl": "2.9.0",
```

Comment:
Seems like we should retain the `^` unless there's a reason to pin to this specific version.

Reply:
> Seems like we should retain the `^` unless there's a reason to pin to this specific version.

The EUA and admin-console is using this specific version. Should we be consistent with them?

Reply:
I think we want to be a bit more permissive with supported peer dependency versions, since this is a shared library. If a consuming application wants to use 2.9.1, it seems like they should be able to do so without getting a yarn warning.